library/core/src/macros/mod.rs

```diff
@@ -460,8 +460,8 @@ macro_rules! r#try {
 /// import the traits qualified so their names do not conflict:
```

Comment:
This sentence no longer reflects what's written in the code block below. Not sure how to phrase it though. Some suggestions:

```suggestion
/// import the traits:
```

```suggestion
/// import the traits qualified binding each to `_` or to a unique name so their names do not conflict:
```

Reply:
@fmease I think `as _` is still "import the traits qualified", but nonetheless I can change this.